shared/Microsoft.Extensions.TypeNameHelper.Sources/TypeNameHelper.cs

```diff
@@ -129,5 +143,18 @@ private static void ProcessGenericType(StringBuilder builder, Type type, Type[]
             }
             builder.Append('>');
         }
+
+        private struct DisplayNameOptions
```

Comment:
How about using named Tuple?

Reply:
Sure.

Reply:
Well, the intent of this was to avoid changing all of these methods if we ever do add an extra option. Tuple types wouldn't really help in that case.

Reply:
I would say its YAGNI.
Further all the methods which would be updated are private so non-breaking.
I don't have strong opinion about it. I will leave it upto you.